### PR TITLE
Relay example: use SX Design

### PR DESCRIPTION
**Pull request by mrtnzlml**
There are many other parts of the application that could be written
better and with more of the SX Design. However, this should be a good
start.

### CODE REVIEW

File: src/example-relay/src/Homepage/locations/LocationsPaginated.js [hunk 1]
```diff
@@ -52,7 +52,8 @@ export default function LocationsPaginated(props: Props): Node {
           <Location key={edge?.node?.id} location={edge?.node} />
         ))}
       </LocationList>
-      <Button onClick={loadMore} disabled={!hasNext}>
+      <Button onClick={loadMore} isDisabled={!hasNext}>
+        {/* $FlowFixMe[incompatible-type]: Button children should be FBT */}
```
Comment (mrtnzlml) [inline src/example-relay/src/Homepage/locations/LocationsPaginated.js:56]: We need to think about this type in SX Design. The goal is to encourage well-built (and therefore translated) applications so maybe we can use here FBT as well. 🤔

Comment (michalsanger) [reply src/example-relay/src/Homepage/locations/LocationsPaginated.js:56]: That tight coupling with FBT?

**Review by michalsanger — APPROVED** [review]

File: src/example-relay/src/Homepage/locations/LocationsPaginated.js (hunk 1 above)
Comment (mrtnzlml) [reply src/example-relay/src/Homepage/locations/LocationsPaginated.js:56]: I will relax the `Button` types in a follow-up PR.

Comment (mrtnzlml) [reply src/example-relay/src/Homepage/locations/LocationsPaginated.js:56]: https://github.com/adeira/universe/pull/2877

**Pull request merged**